examples/Microsoft.Spark.CSharp.Examples/MachineLearning/SentimentStream/Program.cs

```diff
@@ -0,0 +1,107 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.ML;
+using Microsoft.ML.Data;
+using Microsoft.Spark.Sql;
+using Microsoft.Spark.Sql.Streaming;
+
+namespace Microsoft.Spark.Examples.MachineLearning.SentimentStream
+{
+    /// <summary>
+    /// Example of using ML.NET + .NET for Apache Spark
+    /// for sentiment analysis of streaming data.
+    /// </summary>
+    internal sealed class Program : IExample
+    {
+        public void Run(string[] args)
+        {
+            if (args.Length != 3)
+            {
+                Console.Error.WriteLine(
+                    "Usage: SentimentAnalysisStream <host> <port> <model path>");
+                Environment.Exit(1);
+            }
+
+            // Create Spark Session
+            SparkSession spark = SparkSession
+                .Builder()
+                .AppName("Streaming Sentiment Analysis")
+                .GetOrCreate();
+
+            // Setup stream connection info
+            string hostname = args[0];
+            var port = args[1];
+
+            // Read streaming data into DataFrame
+            DataFrame words = spark
+               .ReadStream()
+               .Format("socket")
+               .Option("host", hostname)
+               .Option("port", port)
+               .Load();
+
+            // Use ML.NET in a UDF to evaluate each incoming entry
+            spark.Udf().Register<string, bool>(
+                "MyUDF",
```

Comment:
Batch example uses this:
```suggestion
                "MLudf",
```